website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,539 @@
+---
+layout: docs
+page_title: Manage custom login options
+description: >-
+  Use custom login flows to control how clients can authenticate to Vault
+  through the GUI.
+---
+
+# Manage custom login options for the Vault GUI
+
+@include 'alerts/enterprise-and-hcp.mdx'
+
+Use login customization to set preferred authentication methods and control which methods display in the Vault GUI login form.
+
+## Before you start 
+
+@include 'alerts/restricted-admin.mdx'
+
+- **You must have Vault Enterprise 1.20.0 or higher installed.**
+- **You must have the appropriate permissions**:
+  - You must have `update` permission for the [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth) endpoint.
+ 
+## Step 1: Select your authentication method
+
+Only the following methods are supported by the GUI and can be set as a default or backup method:
+
+@include 'ui/supported-login-methods.mdx'
+
+You can customize your login flow with a default authentication method,
+preferred authentication methods, or both.
+
+Configuration           | Result
+----------------------- | ------
+Default only            | Clients must use the selected authentication method during login.
+Preferred only          | Client must choose from the selected authentication methods during login.
+Default and preferred   | Clients see the default method first but can access the preferred methods as backup methods.
+
+
+## Step 2: Make the authentication method visible to the GUI
+
+Set the listing visibility of your authentication plugin to `unauth` so Vault
+knows to make it available in the GUI login flow. 
```

Comment:
I think this could probably be reworded to the following to make it clearer. 
```suggestion
Set the listing visibility of your authentication plugin to `unauth` so the Vault GUI can build the correct URL for authenticating the user.
```